pkg/ovs/ovsctl/appctl_test.go

```diff
@@ -110,3 +125,135 @@ IPv6 ND Extension: No`,
 		})
 	}
 }
+
+func TestTrace(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
Why add this?

Reply:
This was suggested buy @rajnkamr here https://github.com/antrea-io/antrea/pull/4517#discussion_r1059880934